### ISSUE TITLE
LoraType for sale?

### ISSUE THREAD

**Issue by carlos1w** [AutomationArt/LoraType#1]
Hello there!
This is a neat device.  I was wondering if it is possible to support your efforts by purchasing some boards or devices?  
Thanks

**Comment by jowdow** [comment]
> Hello there! This is a neat device. I was wondering if it is possible to support your efforts by purchasing some boards or devices? Thanks

I second this

**Comment by gooddisplayepaper** [comment]
> Hello there. Nice project. I am Freddie ([freddie@good-display.com](mailto:freddie@good-display.com)). If possible can we talk more about it? Very much like to offer better support. https://twitter.com/Frrrrrrreddie

> I third this. I personally will buy one if you are offering any! I mean it. GDEH0154D67 can also be substuted with GDEY0154D67, completely compatible only a different FPC. And it has fast update so better user experience.

**Comment by MaxSoloAT** [comment]
> Hello there! This is a neat device. I was wondering if it is possible to support your efforts by purchasing some boards or devices? Thanks

Hi) The kit consists of three devices. This allows you to test the mesh. The price is $270 per set. We will be able to send it after April 7.

**Comment by MaxSoloAT** [comment]
> > Hello there. Nice project. I am Freddie ([freddie@good-display.com](mailto:freddie@good-display.com)). If possible can we talk more about it? Very much like to offer better support. https://twitter.com/Frrrrrrreddie
> 
> > I third this. I personally will buy one if you are offering any! I mean it. GDEH0154D67 can also be substuted with GDEY0154D67, completely compatible only a different FPC. And it has fast update so better user experience.

Hi) In the future, we want to refuse the connector and solder the display to the board. "GDEY0154D67 has fast update" sounds really nice. Can we get it?

**Comment by carlos1w** [comment]
> Hi) The kit consists of three devices. This allows you to test the mesh. The price is $270 per set. We will be able to send it after April 7.
Thanks
Let us know when this is available.

**Comment by carlos1w** [comment]
> 
> Hi) The kit consists of three devices. This allows you to test the mesh. The price is $270 per set. We will be able to send it after April 7.

Thanks
Let us know when this is available.

**Comment by gooddisplayepaper** [comment]
> > > Hello there. Nice project. I am Freddie ([freddie@good-display.com](mailto:freddie@good-display.com)). If possible can we talk more about it? Very much like to offer better support. https://twitter.com/Frrrrrrreddie
> > 
> > 
> > > I third this. I personally will buy one if you are offering any! I mean it. GDEH0154D67 can also be substuted with GDEY0154D67, completely compatible only a different FPC. And it has fast update so better user experience.
> 
> Hi) In the future, we want to refuse the connector and solder the display to the board. "GDEY0154D67 has fast update" sounds really nice. Can we get it?

Yes, you have email for me or we can talk on twitter? This fast update samples is in stock. Glad to offer if you need some

**Comment by carlos1w** [comment]
any update?
